packages/web-components/src/components/ic-text-field/ic-text-field.tsx

```diff
@@ -374,6 +358,31 @@ export class TextField {
     }
   }
 
+  private getMaxLengthExceeded = (value: string) => {
+    this.numChars = value.length;
+
+    if (this.type === "number") {
+      if (value && Number(value) < Number(this.min)) {
```

Comment:
these could be shortened to 
`this.minValueUnattained = ((value && Number(value) < Number(this.min)) ? true : false;`
i think?